app/validators/content_item_uniqueness_validator.rb

```diff
@@ -6,14 +6,14 @@ def validate(record)
     unique_fields = Queries::ContentItemUniqueness.unique_fields_for_content_item(content_item)
 
     return unless unique_fields
+    return unless content_item.requires_base_path?
```

Comment:
I don't think we want this line here, as I expect this will mean we don't catch instances where a contact has a base_path and it collides with a different contact content_item.